examples/viewer.html

```diff
@@ -590,9 +621,29 @@
           false).start();
       addShowContainer('7');
 
-      showContainer('1');
+      new Viewer(
+            'container8',
+            '8',
+            '../test/manual/amp-carousel-0-2/vertical.amp.html?amp_js_v=0.1',
+            false).start();
+        addShowContainer('8');
+
+      if (window.location.search) {
+        new Viewer(
+            'container-dynamic',
+            'dynamic',
+            `${window.location.search.slice(1)}?amp_js_v=0.1`,
```

Comment:
This is a bit confusing to me, is the plan here to allow people to pass a whole uri with extra params that should get passed along to the page?

For example, should I be able to do some.url/?some.page%3Ffoo=bar

Either way, I think it would be better to be explicit here to allow for other URL params in the future and look for a specific URL param's value rather than taking the whole search string.

Reply:
I changed this to behave like this PWA `#href=...` hash param